rust/datafusion/src/physical_plan/expressions.rs

```diff
@@ -1782,11 +1804,13 @@ mod tests {
 
     // runs an end-to-end test of physical type coercion:
     // 1. construct a record batch with two columns of type A and B
+    //  (*_ARRAY is the Rust Arrow array type, and *_TYPE is the DataType of the elements)
```

Comment:
Unrelated to this PR, I just renamed some of these arguments to match the comment description